test/e2e/tinkerbell_test.go

```diff
@@ -1932,3 +1932,18 @@ func TestTinkerbellAirgappedKubernetes127UbuntuProxyConfigFlow(t *testing.T) {
 
 	runTinkerbellAirgapConfigProxyFlow(test, "10.80.0.0/16")
 }
+
+func TestTinkerbellKubernetes127UbuntuCuratedPackagesClusterAutoscalerSimpleFlow(t *testing.T) {
+		minNodes := 1
+		maxNodes := 2
+		test := framework.NewClusterE2ETest(t,
+			framework.NewTinkerbell(t, framework.WithUbuntu127Tinkerbell()),
+			framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube127), api.WithWorkerNodeAutoScalingConfig(minNodes, maxNodes)),
+			framework.WithControlPlaneHardware(1),
```

Comment:
why need to set cp node number? I just don't see it else where so if it's default to 1?

Reply:
I was following other packages examples here. Searching for `WithControlPlaneHardware(1` I do see some other examples, particularly for packages but also for some other types of tests.

I kind of prefer it explicit like this, but if you'd rather I take it out I can do that, just let me know.